x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/core/security/transport/netty4/SecurityNetty4Transport.java

```diff
@@ -150,18 +150,16 @@ protected ChannelHandler getClientChannelInitializer(DiscoveryNode node, Connect
     }
 
     @Override
-    protected Bootstrap getClientBootstrap(ConnectionProfile connectionProfile) {
-        final Bootstrap bootstrap = super.getClientBootstrap(connectionProfile);
-        if (false == REMOTE_CLUSTER_PROFILE.equals(connectionProfile.getTransportProfile())
-            || remoteClusterClientBootstrapOptions.isEmpty()) {
-            return bootstrap;
-        }
-
-        logger.trace("reconfiguring client bootstrap for remote cluster client connection");
-        // Only client connections to a new RCS remote cluster can have transport profile of _remote_cluster
-        // All other client connections use the default transport profile regardless of the transport profile used on the server side.
-        remoteClusterClientBootstrapOptions.configure(bootstrap);
-        return bootstrap;
```

Comment:
We still need this for the client channel. The tcp settings do not have separate ones for client and server. Some of them are meaningful for client channels, e.g. `tcp.send_buffer_size`.

Reply:
Good catch, this was a mistake.